my-app/src/pages/Calendar/components/DeliveryCard.module.css

```
@@ -105,14 +108,26 @@
   font-weight: 600;
-  font-size: 1.25rem;
+  font-size: 1.15rem;
+  line-height: 1.2;
   cursor: pointer;
+  display: block;
+  flex: 1;
+  width: 100%;
+  max-width: 100%;
+  min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)